content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -330,21 +353,23 @@ data:
     secret.code.lives=30
 ```
 
-#### Create ConfigMaps from literal values
+### Create ConfigMaps from literal values
```

Comment:
```suggestion
### Create ConfigMap objects from literal values
```

Reply:
We should take care to preserve existing hyperlinks. Also I think I prefer the original heading anyway here.